### PR TITLE
Replaced getpartymember / getguildmember parameter type with constant.

### PR DESCRIPTION
**Pull request by emistry**
constant for getpartymember ( <party id>,<type> )
constant for getguildmember ( <guild id>,<type> )

### CODE REVIEW

**Comment by emistry** [comment]
lol.. =='' how come all previous changes was commited.. ==''

File: db/const.txt [hunk 1]
```diff
@@ -3431,3 +3431,14 @@ SEPTEMBER	9
...
+
+// getpartymember( <party id>,<type> );
+PARTYMEMBER_NAME	0
+PARTYMEMBER_CID	1
+PARTYMEMBER_AID	2
+
+// getguildmember( <guild id>,<type> );
+GUILDMEMBER_NAME	0
+GUILDMEMBER_CID	1
+GUILDMEMBER_AID	2
```
Comment (Smokexyz) [inline db/const.txt]: Cant they both share member name , cid and aid constants since the types are the same?

Comment (dastgir) [inline db/const.txt]: For these 2 commands, they can... Have a command name maybe ?

Comment (emistry) [inline db/const.txt]: personally i would prefer
```
MEMBER_AID
MEMBER_CID
MEMBER_NAME
```
or something even shorter ... maybe =D

**Comment by Jedzkie** [comment]
@emistry suggestion

PMEM_
GMEM_

**Pull request closed without merging**